analyzer/queries.go

```diff
@@ -439,12 +439,12 @@ func (qf QueryFactory) RuntimeTokenChangeUpdateQuery() string {
 
 func (qf QueryFactory) RefreshDailyTxVolumeQuery() string {
 	return `
-		REFRESH MATERIALIZED VIEW daily_tx_volume
+		REFRESH MATERIALIZED VIEW stats.daily_tx_volume
```

Comment:
hm and stats too. I'd better ask in slack

Reply:
Same as `multichain`. Could be in `multichain`, but we'll continue to have more stats tables so I feel it's nice to carve out a schema for them now.